tests/scripts/create-dev-cluster.sh

```diff
@@ -101,41 +128,45 @@ check_examples_dir() {
 }
 
 wait_for_rook_operator() {
-    echo "Waiting for rook operator"
-    $KUBECTL rollout status deployment rook-ceph-operator -n rook-ceph --timeout=180s
-    while ! $KUBECTL get cephclusters.ceph.rook.io -n rook-ceph -o jsonpath='{.items[?(@.kind == "CephCluster")].status.phase}' | grep -q "Ready"; do
-	echo "Waiting for cluster to be ready..."
+    echo "Waiting for rook operator..."
+    $KUBECTL rollout status deployment rook-ceph-operator -n "$ROOK_OPERATOR_NS" --timeout=180s
+    while ! $KUBECTL get cephclusters.ceph.rook.io -n "$ROOK_CLUSTER_NS" -o jsonpath='{.items[?(@.kind == "CephCluster")].status.phase}' | grep -q "Ready"; do
+	echo "Waiting for ceph cluster to become ready..."
 	sleep 20
     done
 }
 
 enable_rook_orchestrator() {
     echo "Enabling rook orchestrator"
-    $KUBECTL rollout status deployment rook-ceph-tools -n rook-ceph --timeout=30s
-    $KUBECTL -n rook-ceph exec -it deploy/rook-ceph-tools -- ceph mgr module enable rook
-    $KUBECTL -n rook-ceph exec -it deploy/rook-ceph-tools -- ceph orch set backend rook
-    $KUBECTL -n rook-ceph exec -it deploy/rook-ceph-tools -- ceph orch status
+    $KUBECTL rollout status deployment rook-ceph-tools -n "$ROOK_CLUSTER_NS" --timeout=90s
```

Comment:
@rkachach , This change from 30s to 90s timeout seems unrelated to the introduction of cmdline parameters for namespaces. Shouldn't it therefore be a separate commit and maybe even a separate PR?

Reply:
@obnoxxx you are right, I'll put it as  separate commit because it's very small change and doesn't deserve its own PR :)